.env

```
@@ -5,7 +5,7 @@ VITE_IDP_SCOPE="openid profile email lakekeeper"
 VITE_IDP_RESOURCE=""
 VITE_IDP_POST_LOGOUT_REDIRECT_PATH="/logout"
 VITE_IDP_TOKEN_TYPE="access_token"
-VITE_ENABLE_AUTHENTICATION=true
-VITE_ENABLE_PERMISSIONS=true
+VITE_ENABLE_AUTHENTICATION=false
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_ENABLE_AUTHENTICATION key should go before the VITE_IDP_AUTHORITY key

(UnorderedKey)